$OpenBSD: patch-tests_run_test_lua,v 1.1 2020/11/12 14:39:54 semarie Exp $
Remove git usage in tests. The tarball isn't a git checkout and the output could be too verbose if /usr/ports in a git checkout.

Index: tests/run_test.lua
--- tests/run_test.lua.orig
+++ tests/run_test.lua
@@ -186,25 +186,6 @@ if not pcall(require, "luassert") then
     os.exit(1)
 end
 
--- Check for untracked files in Git
-do
-    local untracked = {}
-    local f = io.popen("git ls-files --others --exclude-standard")
-    for line in f:lines() do
-        table.insert(untracked, line)
-    end
-    f:close()
-
-    if #untracked > 0 then
-        local c_yellow = string.char(27) .. "[0;33m"
-        local c_reset = string.char(27) .. "[0;0m"
-        print(c_yellow .. "WARN" .. c_reset .. " The following files are untracked:")
-        for _, line in ipairs(untracked) do
-            print("  " .. line)
-        end
-    end
-end
-
 -- Find a free server number
 -- Does have a race condition...
 for i=0,math.huge do
